Logger/LogFactory.cs

```diff
@@ -1,11 +1,24 @@
-﻿namespace Logger
-{
+﻿namespace Logger;
+
     public class LogFactory
     {
-        public BaseLogger CreateLogger(string className)
+        private string? _filePath;
```

Comment:
```suggestion
        private string? _FilePath;
```
Make sure you use pascal casing for these